deployment_scripts/deploy_windows.ps1

```diff
@@ -44,9 +71,12 @@ function Deploy-Windows([String] $monkey_home = (Get-Item -Path ".\").FullName,
     try
     {
         $version = cmd.exe /c '"python" --version  2>&1'
-        if ( $version -like 'Python 3.*' ) {
+        if ($version -like 'Python 3.*')
+        {
             "Python 3.* was found, installing dependencies"
-        } else {
+        }
+        else
+        {
             throw System.Management.Automation.CommandNotFoundException
```

Comment:
Add which command was missing in string param of exception constructor

Reply:
I don't understand this comment

Reply:
@ShayNehmad we catch the exception later, so not necessary